lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Screens/BaseClasses/SearchPosts/LMFeedBaseSearchPostViewModel.swift

```diff
@@ -40,7 +40,7 @@ public class LMFeedBaseSearchPostViewModel {
         self.delegate = delegate
         self.postList = []
         self.page = 1
-        self.pageSize = 10
+        self.pageSize = 20
```

Comment:
Was it intentional or was for testing purpose ?

Reply:
It was intentional